io_scene_dae/export_dae.py

```diff
@@ -61,27 +61,26 @@ def snap_tup(tup):
 
 
 def strmtx(mtx):
-    s = " "
+    s = ""
     for x in range(4):
         for y in range(4):
-            s += str(mtx[x][y])
-            s += " "
-    s += " "
+            s += " {}".format(mtx[x][y])
+    s = " {} ".format(s)
     return s
```

Comment:
If you can squash it into ef27363f (maybe fixing the authorship at the same time? GitHub doesn't recognize you as author for this one) it would be perfect, but not mandatory either, I can merge as is.